docs/changes.rst

```diff
@@ -82,9 +80,14 @@ Released: not yet
   qualifiers.  In this case, the property value is defined in the MOF as
   a string or array of strings that compiles to a CIMInstance.  This
   change does not compile CIMClass definitions.
-  Originally these compiled objects were passed through the compiler as 
+  Originally these compiled objects were passed through the compiler as
   strings. (See issue # 2277).
 
+* Mock support: Added a method BaseProvider.is_subclass() that tests whether
```

Comment:
I would move this one to enhancements or something. It is not really a bug

Reply:
It is already in section Enhancements.